app/views/layouts/_header.html.erb

```diff
@@ -0,0 +1,113 @@
+<nav class="navbar navbar-expand-lg navbar-dark"
+      style='background-color: #00401F;' aria-label="Fourth navbar example">
+  <div class="container">
+    <div class="navbar-brand">
+      <%= link_to image_tag(@organization_settings[:logo], class: 'logo me-2', alt: 'site logo'),
```

Comment:
```suggestion
      <%= link_to image_tag(Current.organziation.logo, class: 'logo me-2', alt: 'site logo'),
```
An idea of how to pass around the organization to make tweaks on the UI.

Reply:
or `current_tenant.logo` ?